### ISSUE TITLE
Fixed column width for bottom menu

### ISSUE THREAD

**Issue by titojankowski** [giving-a-fuck-about-climate-change/carbon-inferno#80]
Fix the width according to the original sketch mockup. Currently the spacing expands based on the window size, should be fixed/stay the same.

**Comment by titojankowski** [comment]
This is for the “About the project”, “Social”, and “Give a fuck” columns

**Comment by stephanieinez** [comment]
Hey @titojankowski I don't understand this issue?

**Comment by titojankowski** [comment]
hi @stephanieinez !!!
Let me know if this helps. The footer columns are "tighter" in the mockup. In the current version the columns spread out based on the window size, so if the window is big the columns are really far apart. To me this means to set the width of the columns rather than let them grow based on the window size. That said, that's not stated explicitly in the mockup, so perhaps I'm interpreting it wrong.

Mockup: 
![screen shot 2017-11-16 at 4 44 53 pm](https://user-images.githubusercontent.com/129060/32923646-90214de6-caed-11e7-9a0d-a1f06e338c9e.png)

Current version:
![screen shot 2017-11-16 at 4 45 40 pm](https://user-images.githubusercontent.com/129060/32923666-a3ba2198-caed-11e7-9cfd-324be8ce1b05.png)

**Comment by stephanieinez** [comment]
Ah okay. I personally think it doesn't look very good when I add a fixed width to the columns - it looks a lot cleaner when they fit the page. I think their position should be relative to the width of the chart above it - keeps the page balanced. What do you think?

**Comment by titojankowski** [comment]
I checked out 3 sites for comparison to add ideas:

Coinbase <https://www.coinbase.com/>: columns expand to fit the page. But
they have Each is about 130 - 160 pixels wide.
Crunchbase <https://crunchbase.com/>: Fixed width, 170px per column, float
in the middle.
Angelist <https://angel.co/>: Fixed width, center all the menus in the
middle, ~200px.

Our columns are much further apart than any of these sites. Being so wide
apart makes them seem like different functions or something, rather than
one set of navigation.

How about fixing the width and floating them in the center, does that help
it balance out?

Can also ask Purin, these sort of questions are his speciality.

Tito




On Fri, Nov 17, 2017 at 8:52 AM, Stephanie Tassone <notifications@github.com
